patches/libgpg-error/0001-Patch-out-a-broken-operation-during-cross-compilatio.patch

```diff
@@ -0,0 +1,26 @@
+From 02eb5a33aa55e56fe07398cafaad2778c8fabcfc Mon Sep 17 00:00:00 2001
+From: Dennisbonke <admin@dennisbonke.com>
+Date: Sun, 27 Dec 2020 21:21:40 +0100
+Subject: [PATCH] Patch out a broken operation during cross compilation
+
+Signed-off-by: Dennisbonke <admin@dennisbonke.com>
+---
+ src/Makefile.am | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/Makefile.am b/src/Makefile.am
+index fc3acc3..d58871a 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -353,7 +353,7 @@ gpg-error-config: gpgrt-config gpg-error-config-old
+ 	else                                                   \
+ 	  echo "no";                                           \
+ 	  echo "***  Please report to <https://bugs.gnupg.org> with gpg-error-config-test.log"; \
+-	  exit 1; \
++	  #exit 1; \
```

Comment:
Why is this done?

Reply:
`libgpg-error` tries to execute the newly compiled binary, which fails because it is compiled for managarm. As running it is just a sanity check from `libgpg-error`, I removed the failure out of it so the build succeeds